.circleci/config.yml

```diff
@@ -44,7 +47,7 @@ commands:
     - run:
         name: Install system requirements
         command: |
-          pip3 install awscli==$AWSCLI_VERSION --upgrade
+          pip install awscli==$AWSCLI_VERSION --upgrade
```

Comment:
I'm assuming `pip` is aliased to `pip3` here?

Reply:
yep!